### PR TITLE
changes to copy_consensus focal_cn and tp53 status 

### PR DESCRIPTION
**Pull request by runjin326**
### Purpose

_Briefly describe your changes here._
The following changes are included:
1) description of 'deep deletion' and `amplification` for focal-cn module 
2) description of changes in consensus calling 
3) description of TP53 `loss` and `activated` status calling.

### Issue

_What GitHub issue does your pull request address?_
https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/132
https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/133
https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/137

#### Which areas should receive a particularly close look?
Please see whether the descriptions are clear and easy-to-read.

#### Is there anything that you want to discuss further?
No

#### Is the pull request ready for review?
Yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Review by jaclyn-taroni — COMMENTED** [review]
Thanks so much for submitting this @runjin326! This looks good from a content point of view – most of the comments I am returning have to do with tweaks to wording, style, or structure.

From this comment https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/145#issuecomment-901414234, we can see that there are multiple potential spelling errors. To resolve those issues, please follow the spellcheck step instructions in [the pull request text](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/.github/PULL_REQUEST_TEMPLATE.md#spellcheck-step) to add and fix spelling errors – it looks like Li-Fraumeni syndrome is misspelled in one place.

In the future, I'd recommend limiting pull requests to a single issue or multiple issues if they're all related to the same section (e.g., #132 and #133 could get addressed in the same PR, but #137 might be submitted separately). Smaller reviews focused on a single area are easier to return in a timely manner. 

Thanks again! Please let me know if you have any questions or if any of my comments are unclear.

File: content/03.methods.md [hunk 1]
```diff
@@ -384,6 +388,16 @@ Additional details can be found in the analysis [notebook](https://alexslemonade
 6. Non-MB and non-ATRT embryonal tumors with <i>CIC-NUTM1</i> or other <i>CIC</i> fusions, were subtyped as `CNS EFT-CIC` (CNS Ewing sarcoma family tumor with <i>CIC</i> alteration).
 7. Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as CNS Embryonal, NOS (CNS Embryonal tumor, not otherwise specified).
 
+In addition to tumor types mentioned above, TP53 altered status is also annotated for all samples and if a sample is determined to be either `TP53 loss` or `TP53 activated`, this annotation will be included in the `molecular_subtype` column.
```
Comment (runjin326) [inline content/03.methods.md:391]:
```suggestion
#### TP53 Alteration Annotation
In addition to tumor types mentioned above, TP53 altered status is also annotated for all samples and if a sample is determined to be either `TP53 loss` or `TP53 activated`, this annotation will be included in the `molecular_subtype` column. 
```

File: content/03.methods.md [hunk 2]
```diff
@@ -384,6 +386,16 @@ Additional details can be found in the analysis [notebook](https://alexslemonade
 6. Non-MB and non-ATRT embryonal tumors with <i>CIC-NUTM1</i> or other <i>CIC</i> fusions, were subtyped as `CNS EFT-CIC` (CNS Ewing sarcoma family tumor with <i>CIC</i> alteration).
 7. Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as CNS Embryonal, NOS (CNS Embryonal tumor, not otherwise specified).
 
+In addition to tumor types mentioned above, TP53 altered status is also annotated for all samples and if a sample is analyzed to be either `TP53 loss` or `TP53 activated`, this annotation will be included in the `molecular_subtype` column.
```
[the remark this answers is not in the document]
Comment (runjin326) [reply content/03.methods.md]: Hello @jaclyn-taroni, thanks so much for making those modifications! I will separate edits that are not related into separate PRs next time - and also do the spelling check!
For the TP53 section, what do you think about adding `#### TP53 Alteration Annotation` as I suggested below?

**Review by jaclyn-taroni — APPROVED** [review]
Looks good to me, thank you for the updates! Once we add the new header, we can get this merged!

**Pull request merged**